tango/step.py

```diff
@@ -79,6 +79,11 @@ class StepResources(FromParams):
     Minimum number of GPUs. It must be non-negative.
     """
 
+    gpu_type: Optional[str] = None
+    """
+    The type of GPU that the step requires, e.g. 'NVIDIA A100-SXM-80GB'.
```

Comment:
Where do these strings come from? Can I see a list of them somewhere?

Reply:
It depends on the executor. For the Beaker executor this should match what you see in the Beaker UI.